doc-site/docs/reference/types/_includes/contractlistener_description.md

```diff
@@ -4,3 +4,100 @@ of the interface for that event.
 
 Check out the [Custom Contracts Tutorial](../../tutorials/custom_contracts/index.md) for
 a walk-through of how to set up listeners for the events from your smart contracts.
+
+See below for a deep dive into the format of contract listeners and important concepts to understand when managing them.
+
+### Multiple filters
+
+From v1.3.1 onwards, a contract listener can be created with multiple filters under a single topic when supported by the connector.
+
+Before this change, each contract listener would only support listening to one specific event from an interface previously defined. Each listener would be comprised of:
+
+- a reference to a specific blockchain event to listen for
+- (optional) a specific location/address to listen from
+- a connector-specific signature (generated from the event), which allows you to easily identify and search for the contact listener for an event
+- a `topic` which determines the ordered stream that these events are part of
+
+This format is still supported by the API. However, it may not fully guarantee accurate ordering of events coming from multiple listeners, even if they share the same `topic` (such as during cases of blockchain catch-up, or when one listener is created much later than the others).
```

Comment:
```suggestion
For backwards compatibility the `event` and `signature` fields are still supported by the API. They contain the `event` from the first `filters` entry.
```

I _think_ this is the point being made here? Not sure. Overall this paragraph seems a little confusing.

Reply:
No, you've made that point below.

I think this paragraph just needs removing

Reply:
> it may not fully guarantee accurate ordering of events coming from multiple listeners

This statement has nothing to do with the API input/output format. I think you've elsewhere made the point how the new flexibility of allowing `len(filter) > 1` is a good thing.